modules/machineset-yaml-ibm-cloud.adoc

```diff
@@ -0,0 +1,131 @@
+ifndef::infra[`node-role.kubernetes.io/<role>: ""`.]
+ifdef::infra[`node-role.kubernetes.io/infra: ""`.]
+
+In this sample, `<infrastructure_id>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and
+ifndef::infra[`<role>`]
+ifdef::infra[`<infra>`]
+is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+ifndef::infra[]
+    machine.openshift.io/cluster-api-machine-role: <role> <2>
+    machine.openshift.io/cluster-api-machine-type: <role> <2>
+  name: <infrastructure_id>-<role>-<region> <3>
+endif::infra[]
+ifdef::infra[]
+    machine.openshift.io/cluster-api-machine-role: <infra> <2>
+    machine.openshift.io/cluster-api-machine-type: <infra> <2>
+  name: <infrastructure_id>-<infra>-<region> <3>
+endif::infra[]
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+ifndef::infra[]
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<region> <3>
+endif::infra[]
+ifdef::infra[]
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<infra>-<region> <3>
+endif::infra[]
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+ifndef::infra[]
+        machine.openshift.io/cluster-api-machine-role: <role> <2>
+        machine.openshift.io/cluster-api-machine-type: <role> <2>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<region> <3>
+endif::infra[]
+ifdef::infra[]
+        machine.openshift.io/cluster-api-machine-role: <infra> <2>
+        machine.openshift.io/cluster-api-machine-type: <infra> <2>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<infra>-<region> <3>
+endif::infra[]
+    spec:
+      metadata:
+        labels:
+ifndef::infra[]
+          node-role.kubernetes.io/<role>: ""
+endif::infra[]
+ifdef::infra[]
+          node-role.kubernetes.io/infra: ""
+endif::infra[]
+      providerSpec:
+        value:
+          apiVersion: ibmcloudproviderconfig.openshift.io/v1beta1
+          credentialsSecret:
+            name: ibmcloud-credentials
+          image: <infrastructure_id>-rhcos <4>
+          kind: IBMCloudMachineProviderSpec
+          primaryNetworkInterface:
+              securityGroups:
+              - <infrastructure_id>-sg-cluster-wide
+              - <infrastructure_id>-sg-openshift-net
+              subnet: <infrastructure_id>-subnet-compute-<region>-<zone> <5>
+          profile: <instance_profile> <6>
+          region: <region> <7>
+          resourceGroup: <resource_group> <8>
+          userDataSecret:
+              name: <role>-user-data <2>
+          vpc: <vpc_name> <9>
+          zone: <region>-<zone> <10>
```

Comment:
region is eu-gb, zone is eu-gb-1. so maybe we can remove region for <5> and <10>
```
$ oc get machine                                                                                                                                 
NAME                                PHASE     TYPE       REGION   ZONE      AGE
mihuangfeb25-6c56n-master-0         Running   bx2-4x16   eu-gb    eu-gb-1   50m
```

```
            subnet: mihuangfeb25-6c56n-subnet-compute-eu-gb-1
          profile: bx2-4x16
          region: eu-gb
          resourceGroup: mihuangfeb25-6c56n
          userDataSecret:
            name: worker-user-data
          vpc: mihuangfeb25-6c56n-vpc
          zone: eu-gb-1
```

Reply:
done, thanks!